<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="apply_title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>832</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사물함 신청</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="locker_list">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>431</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="locker_list_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>신청 가능한 사물함</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="applyBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>신청하기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>학생 이름 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="locker_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>선택한 사물함 :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
